godot_export_manager.py

```diff
@@ -318,15 +318,11 @@ def execute(self, context):
 
         path = group[self.idx].export_path
         if (path.find("//") == 0 or path.find("\\\\") == 0):
-            # If relative, convert to absolute
```

Comment:
The next line is:

```python
            path = bpy.path.abspath(path)
```

From that call, one can easily understand (or if not, look it up on the [Blender API](https://www.blender.org/api/blender_python_api_2_77_0/bpy.path.html#bpy.path.abspath)) that a relative path is being transformed into a relative. For this reason, if you say it's OK I'm leaving it that way.